Jenkinsfile

```diff
@@ -94,7 +94,7 @@ kind: Pod
 spec:
   containers:
   - name: maven
-    image: maven:3.9.6-eclipse-temurin-21
+    image: maven:3.9.6-eclipse-temurin-17
```

Comment:
This is wrong. 8 should be compiled by 21.

Reply:
Wow, that's wrong indeed. Thanks for catching it!